public/js/addTask.js

```diff
@@ -7,24 +7,25 @@ async function taskFormHandler(event) {
       window.location.toString().split('/').length - 1
```

Comment:
It won't let me add a comment to line 5 so I am adding it here.  I think when we create the task there will not be a volunteer assigned to it.  It will contain NULL in the task table.  Once the volunteer selects the task we will add their req.session.id to that column of the table.

Reply:
Done.